gradle.properties

```diff
@@ -36,6 +36,7 @@ cyclonedxCoreJavaVersion = 3.0.0
 digraphVersion = 1.0
 disklrucacheVersion = 2.0.2
 flexmarkVersion = 0.62.2
+freemarkerVersion = 2.3.30
```

Comment:
It feels a bit odd to me to introduce a new dependency without using it. Why not squash this with the next commit? Esp. as you only explain in the next commit why FreeMaker was chosen.

In fact, I'd also be fine with also squashing with the commit before this one, which introduces the base class, which also does not make much sense without context.

Reply:
Done.